cms/envs/bok_choy_docker.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+"""
+Settings for Bok Choy tests that are used when running Studio in Docker-based devstack.
+"""
+
+# noinspection PyUnresolvedReferences
+from .bok_choy import *  # pylint: disable=wildcard-import
+
+LMS_BASE = '{}:{}'.format(os.environ['BOK_CHOY_HOSTNAME'], os.environ['BOK_CHOY_LMS_PORT'])
+LMS_ROOT_URL = 'http://{}'.format(LMS_BASE)
+
+# Docker does not support the syslog socket at /dev/log. Rely on the console.
```

Comment:
Unfortunately, because these are modifications of an existing setting, a simple import wouldn't cover this.  We'd have to import a function that takes the existing setting as an argument, and then call it.  Given that it would still be duplicated between cms and lms, I'm not sure how much that gains us right now.